scripts/Dockerfile

```diff
@@ -1,5 +1,4 @@
-FROM scratch
-
-COPY ./coder-logstream-kube /coder-logstream-kube
-
-ENTRYPOINT ["/coder-logstream-kube"]
+FROM --platform=$BUILDPLATFORM scratch AS base
+ARG TARGETARCH
```

Comment:
reviewer note: this comes from `--platform`

ref: https://docs.docker.com/build/guide/multi-platform/#platform-build-arguments